src/database/models/superTokenModel.js

```diff
@@ -16,9 +16,17 @@ module.exports = (db) => { return  db.define("supertokens", {
   pic: {
     type: Sequelize.STRING
   },
-  delay: {
+  pppmode:{
     type: Sequelize.INTEGER,
-    defaultValue: 900
+    defaultValue: 1
+  },
```

Comment:
it seems to me that pppmode doesn't belong to the token model. Unlike the other model fields, this is not part of the protocol state, but just a local configuration. If it doesn't cause complications, I'd remove it from the model and just use the config item.

Reply:
This should be in a bigger refactor process